src/components/Accordion/AccordionItem/AccordionItem.jsx

```diff
@@ -7,29 +7,55 @@ import {AccordionContext} from '~/components/Accordion/Accordion.context';
 
 export const AccordionItem = ({id, label, icon, onClickToOpen, onClick, onClickToClose, children, ...props}) => {
     const context = useContext(AccordionContext);
-    const open = context.currentItem === id;
+    let open;
+
+    if (context.isMultipleOpenable) {
+        open = context.currentItemId ? context.currentItemId.includes(id) : false;
+    } else {
+        open = context.currentItemId === id;
+    }
 
     const handleClick = e => {
         if (!open) {
             onClickToOpen(e);
         }
 
         onClick(e);
-        context.defineCurrentItem(id);
+        context.setOpenedItemId(id);
```

Comment:
This one shoudl only call the setOpenedItemId prop from accordion - not modifying any state, if component is controlled